debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -880,6 +880,9 @@ public synchronized Connection connection(boolean executeOnConnect) throws SQLEx
             if (initialOps != null) {
                 execute(initialOps);
             }
+
+            LOGGER.info("Connected to node: {}", getConnectedNode(conn));
```

Comment:
Would the Logger context here also give the name of the connector ? 
The motivation behind this requirement is to correlate the connector_name, the tserver to which it is connected

Reply:
The logger context here would be either `streaming` or `snapshot` so it might not be much useful here.